src/stylesheets/components/global/task/task.scss

```diff
@@ -0,0 +1,78 @@
+.task-container,
+.task-container--expanded {
+  position: relative;
+  width: 100%;
+}
+
+.task-container {
+  height: $task-header-height + 4px;
```

Comment:
does this work without wrapping it in `calc()`?

Reply:
Yeah, surprisingly